external/js/cothority/src/personhood/spawner-instance.ts

```diff
@@ -38,6 +38,10 @@ export default class SpawnerInstance extends Instance {
             .add(this.struct.costCredential.value);
     }
 
```

Comment:
To be consistent with the rest I would comment this function

Reply:
Removed `signupCost`, as it is too specific and not used at all.